lib/router-feed-lib/src/router_rpc_wrapper.rs

```diff
@@ -52,10 +55,21 @@ impl RouterRpcClientTrait for RouterRpcWrapper {
         pubkey: &Pubkey,
         config: RpcProgramAccountsConfig,
     ) -> anyhow::Result<Vec<AccountWrite>> {
-        Ok(
-            get_compressed_program_account_rpc(&self.rpc, &HashSet::from([*pubkey]), config)
-                .await?
-                .1,
-        )
+        let disable_compressed = std::env::var::<String>("DISABLE_COMRPESSED_GPA".to_string())
```

Comment:
this should be a config variable in RpcProgramAccountsConfig not an env variable

Reply:
This method is used only in the smoke tests, tests do not use RpcProgramAccountsConfig. So no use of removing this env variable.